rca/people/models.py

```diff
@@ -24,6 +28,37 @@ def __str__(self):
         return self.title
 
 
+class StaffRole(Orderable):
+    role = models.CharField(max_length=128)
+    programme = models.ForeignKey(
+        "programmes.ProgrammePage",
+        on_delete=models.CASCADE,
+        related_name="related_programme",
+        null=True,
+        blank=True,
+    )
+    custom_programme = models.CharField(
+        max_length=128,
+        help_text=_("Specify a custom programme page here if one does not exist"),
+        blank=True,
+    )
+    page = ParentalKey("StaffPage", related_name="role")
```

Comment:
Since a staff member can have many roles, it might make more sense for the related name to be pluralised here.